src/js/forecast.js

```diff
@@ -0,0 +1,35 @@
+const tempForecastDay1 = document.getElementById('temp-forecast-day1');
+const tempForecastDay2 = document.getElementById('temp-forecast-day2');
+const tempForecastDay3 = document.getElementById('temp-forecast-day3');
+const iconDay1 = document.getElementById('icon-day1');
+const iconDay2 = document.getElementById('icon-day2');
+const iconDay3 = document.getElementById('icon-day3');
+
+function getForecast() {
+  if (localStorage.getItem('location') !== null) {
+    const latitude = JSON.parse(window.localStorage.getItem('latitude'));
+    const longitude = JSON.parse(window.localStorage.getItem('longitude'));
+    fetch(`https://api.openweathermap.org/data/2.5/onecall?lat=${latitude}&lon=${longitude}&exclude=current,minutely,hourly,alerts&appid=0a56a45096652a831cb6980d524fe081&units=metric`)
+      .then((response) => {
+        if (!response.ok) {
+          throw Error(response.statusText);
+        }
```

Comment:
Well, basically if the response would not the 'okay', we would catch the error with the `.catch`.
Do you think we need another response validation here?

Reply:
let's discuss this tomorrow